thcrap_tsa/src/thcrap_tsa.h

```diff
@@ -287,6 +287,7 @@ int patch_msg_dlg(void *file_inout, size_t size_out, size_t size_in, const char
 int patch_msg_end(void *file_inout, size_t size_out, size_t size_in, const char *fn, json_t *patch);
 int patch_end_th06(void *file_inout, size_t size_out, size_t size_in, const char *fn, json_t *patch);
 int patch_anm(void *file_inout, size_t size_out, size_t size_in, const char *fn, json_t *patch);
+size_t anm_get_size(const char* fn, json_t* patch, size_t patch_size);
```

Comment:
This isn't exported, so shouldn't it be in the anm.hpp file?

Reply:
`patch_anm` isn't exported either and it's also in this header file. Declarations for all functions that are passed to `patchhook_register` are in here and none of them are exported

Reply:
Yeah, I wasn't thorough when checking that. Both `patch_msg_dlg` and `patch_msg_end` are exported and I figured the patch prefix meant "patchhook" here. Those exports should probably be removed from the Def file then.